java/com/google/location/lbs/activity/audioset/dataprocessing/incomplete-example.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+option java_outer_classname = "IncompleteExample";
+
+message Example {
+    string ytid = 1;
```

Comment:
not sure if there's a formatting tool, but 2 spaces is the general rule.

Reply:
Ah. Thank you.